set.mm

```diff
@@ -643200,6 +643206,781 @@ base set if and only if the neighborhoods (convergents) of every point
+  $}
+
+  ${
+    simpgnsgd.1 $e |- B = ( Base ` G ) $.
+    simpgnsgd.2 $e |- .0. = ( 0g ` G ) $.
+    simpgnsgd.3 $e |- ( ph -> G e. SimpGrp ) $.
+    $( The only normal subgroups of a simple group are the group itself and the
+       trivial group.  (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    simpgnsgd $p |- ( ph -> ( NrmSGrp ` G ) = { { .0. } , B } ) $=
+      ( cnsg cfv csn c2o cfn wcel cen wbr a1i syl syl2anc cvv cbs cpr 2onn nnfi
+      com simpg2nsg enfii simpggrpd 0idnsgd snex wceq fvex syl6eqel simpgntrivd
+      csimpg neqcomd enpr2d ensymd entr rr-php2d ) ACHIZDJZBUAZAKLMZUTKNOZUTLMA
+      KUDMZVCVEAUBPKUCQACUNMVDGCUEQZUTKUFRABCDEFACGUGUHAVDKVBNOUTVBNOVFAVBKAVAB
+      SSVASMADUIPABCTIZSBVGUJAEPCTUKULABVAABCDEFGUMUOUPUQUTKVBURRUS $.
+  $}
+
+  ${
+    simpgnsg2d.1 $e |- B = ( Base ` G ) $.
+    simpgnsg2d.2 $e |- .0. = ( 0g ` G ) $.
+    simpgnsg2d.3 $e |- ( ph -> G e. SimpGrp ) $.
+    simpgnsg2d.4 $e |- ( ph -> A e. ( NrmSGrp ` G ) ) $.
+    $( A normal subgroup of a simple group is either the whole group or the
+       trivial subgroup.  (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    simpgnsg2d $p |- ( ph -> ( A = { .0. } \/ A = B ) ) $=
+      ( csn cpr wcel wceq wo cnsg cfv simpgnsgd eleqtrd elpri syl ) ABEJZCKZLBU
+      AMBCMNABDOPUBIACDEFGHQRBUACST $.
+  $}
+
+  ${
+    $d ph x $.  $d x .0. $.  $d x B $.  $d x G $.
+    2nsgsimpgd.1 $e |- B = ( Base ` G ) $.
+    2nsgsimpgd.2 $e |- .0. = ( 0g ` G ) $.
+    2nsgsimpgd.3 $e |- ( ph -> G e. Grp ) $.
+    2nsgsimpgd.4 $e |- ( ph -> -. { .0. } = B ) $.
+    2nsgsimpgd.5 $e |- ( ( ph /\ x e. ( NrmSGrp ` G ) ) ->
+      ( x = { .0. } \/ x = B ) ) $.
+    $( If any normal subgroup of a nontrivial group is either the trivial
+       subgroup or the whole group, the group is simple.  (Contributed by Rohan
+       Ridenour, 3-Aug-2023.) $)
+    2nsgsimpgd $p |- ( ph -> G e. SimpGrp ) $=
+      ( wcel wa wceq adantl simpr syl adantr eqeltrd adantlr cvv cfv csn cpr cv
+      cnsg c2o cen wo elprg mpbird cgrp 0nsg nsgid elpri mpjaodan impbida eqrdv
+      wb snex a1i cbs fvexi enpr2d eqbrtrd issimpgd ) ADHADUEUAZEUBZCUCZUFUGABV
+      FVHABUDZVFKZVIVHKZAVJLVKVIVGMZVICMZUHZJVJVKVNURAVIVGCVFUINUJAVKLVLVJVMAVL
+      VJVKAVLLVIVGVFAVLOAVGVFKZVLADUKKZVOHDEGULPQRSAVMVJVKAVMLVICVFAVMOACVFKZVM
+      AVPVQHCDFUMPQRSVKVNAVIVGCUNNUOUPUQAVGCTTVGTKAEUSUTCTKACDVAFVBUTIVCVDVE $.
+  $}
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Classification of abelian simple groups
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    $d ph n $.  $d A n $.
+    1gengend.1 $e |- B = ( Base ` G ) $.
+    1gengend.2 $e |- .x. = ( .g ` G ) $.
+    1gengend.3 $e |- F = ( n e. ZZ |-> ( n .x. A ) ) $.
+    1gengend.4 $e |- ( ph -> A e. B ) $.
+    $( The set of multiples of an element of a group includes that element.
+       (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    1gengend $p |- ( ph -> A e. ran F ) $=
```

Comment:
What is the meaning of the label? "gen" = "generate", "generator"? This should be reflected in the comment somehow.

Reply:
Yes. I've adjusted the comment to reflect this (and also changed the labels to match existing conventions for cyclic subgroups).